src/query/list_file.c

```diff
@@ -5577,12 +5445,7 @@ qfile_delete_list_cache_entry (THREAD_ENTRY * thread_p, void *data)
       return ER_FAILED;
     }
 
-  lent->deletion_marker = true;
-#if defined(SERVER_MODE)
-  if (lent->deletion_marker && lent->last_ta_idx == 0)
-#else /* not SERVER_MODE */
   if (lent->deletion_marker)
```

Comment:
Can 'lent->last_ta_idx == 0' be omitted? Isn't there a transaction using the cache?

Reply:
because the transaction isolation level "REPEATABLE READ" is not need to keep.